test/binCases/no-options/none/stdin.js

```diff
@@ -9,7 +9,8 @@ module.exports = function testAssertions(code, stdout, stderr) {
 	expect(stdout[4]).toContain("");
 	expect(stdout[5]).toContain("WARNING");
 	expect(stdout[6]).toContain("The \'mode\' option has not been set");
-	expect(stdout[7]).toContain("");
-	expect(stdout[8]).toContain("ERROR in Entry module not found");
+	expect(stdout[7]).toContain("You can also set it to \'none\'");
```

Comment:
Where's this from?

Reply:
This is also breaking in other branches. ERROR is being asserted in a different array index, but stdout is correct anyways.

Reply:
This was due to an upstream change that was recently released: https://github.com/webpack/webpack/blob/17034eb61cd70248bf6c1ccbc20d0c6dbe3a34b1/lib/NoModeWarning.js#L18

Our tests were passing by accident, most likely due to caching on the CI servers.